### PR TITLE
Changed database config for backend and other minor bug fix

### LINKED ISSUE #9: use postgres database and serect key and debug should be set from envroronment
See https://github.com/devlup-labs/ta_portal/blob/master/src/ta_portal/settings.py for reference

### PR DESCRIPTION
**Pull request by DarshitJain04**
@Tanuj22 
Fixes #9 
PS : Did not make a separate PR for the eslint config as there was a minor change so pushed it to this PR itself.

### CODE REVIEW

**Comment by Tanuj22** [comment]
CI has to be changed for these changes

File: frontend/src/App.js [hunk 1]
```diff
@@ -5,19 +5,19 @@ import NotFound from './pages/404';
...
-            <Route path="/team" component={Team} />
-            <Route default component={NotFound} />
-          </Switch>
-        </div>
-      </Router>
-    </div>
-  );
+	return (
+		<div className="App">
+			<Router>
```
Comment (Tanuj22) [inline frontend/src/App.js:10]: Why the tab size is so large

Comment (DarshitJain04) [reply frontend/src/App.js:10]: Umm.. I did add `"tabWidth" : 2` in `.prettierrc`

**Comment by Tanuj22** [comment]
CI change needed \\ @Sahil1479

**Review by Tanuj22 — APPROVED** [review]

**Comment by Sahil1479** [comment]
Okay

**Pull request merged**